src/libraries/Microsoft.Extensions.Diagnostics.Abstractions/src/Microsoft.Extensions.Diagnostics.Abstractions.csproj

```
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

nit: for better readability and to make the project file look closer to a `dotnet new classlib` template, please add an empty line after the Project tag.

```suggestion

  <PropertyGroup>
```